golint not found

**Describe the bug**
golint is not found on the github actions runner:
```Run golint -set_exit_status ./...
/home/runner/work/_temp/df7a5e2b-19e9-4b1c-9b18-aad5495[6](https://github.com/liquibase/liquibase-package-manager/runs/7030795143?check_suite_focus=true#step:5:7)ca94.sh: line 1: golint: command not found
Error: Process completed with exit code 12[7](https://github.com/liquibase/liquibase-package-manager/runs/7030795143?check_suite_focus=true#step:5:8).
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to any recent action
2. Click on the build log
4. See error

**Expected behavior**
go code should be linted

**Additional context**
NOTE: Golint is https://github.com/golang/go/issues/38968. There's no drop-in replacement for it, but tools such as [Staticcheck](https://staticcheck.io/) and go vet should be used instead.
https://github.com/golang/lint

--- a/.github/workflows/codeql-analysis.yml
+++ b/.github/workflows/codeql-analysis.yml
@@ -14,7 +14,7 @@ name: "CodeQL"
 on:
   push:
     branches: [ master ]
-  pull_request_target:
+  pull_request:
     # The branches below must be a subset of the branches above
     branches: [ master ]
   schedule:

--- a/.github/workflows/pull-request.yml
+++ b/.github/workflows/pull-request.yml
@@ -16,13 +16,8 @@ jobs:
       - name: Setup GO environment
         run: |
           go mod download
-          go get -u golang.org/x/lint/golint
 
-      - name: Lint
-        run: |
-          golint -set_exit_status ./...
-
-      - name: Unit tests
+      - name: Lint and run unit tests
         run: |
           make test
           go tool cover -func coverage.out

--- a/.github/workflows/pull-request.yml
+++ b/.github/workflows/pull-request.yml
@@ -1,7 +1,7 @@
 name: PR Quality Gate
 
 on:
-  pull_request_target:
+  pull_request:
 
 jobs:
   test:

--- a/Makefile
+++ b/Makefile
@@ -31,14 +31,16 @@ build: updateVersion
 generateExtensionPackages:
 	go run package-manager/cmd/populator
 
-test:
-	golint ./internal/app/...
+test: test-setup
+	staticcheck ./internal/app/...
+	go vet ./internal/app/...
 	go test -v ./internal/app/... -coverprofile=coverage.out -covermode count
 
 cover: test
 	go tool cover -html=coverage.out
 
 test-setup:
+	go install honnef.co/go/tools/cmd/staticcheck@latest
 	curl -Ls https://github.com/mcred/vexrun/releases/download/v0.0.5/vexrun-0.0.5.jar -z $(VEXRUN_FILE) -o $(VEXRUN_FILE)
 
 e2e: test-version test-add test-completion test-help test-install test-list test-remove test-search test-cleanup

--- a/internal/app/commands/root.go
+++ b/internal/app/commands/root.go
@@ -62,7 +62,7 @@ func initConfig() {
 	if err != nil {
 		errors.Exit(err.Error(), 1)
 	}
-	b, err := ioutil.ReadAll(jsonFile)
+	b, _ := ioutil.ReadAll(jsonFile)
 
 	//Load Bytes to Packages
 	packs = app.LoadPackages(b)

--- a/internal/app/packages/Version.go
+++ b/internal/app/packages/Version.go
@@ -113,8 +113,5 @@ func CreateClasspath(cp string) {
 //ClasspathExists checks to see if classpath directory is created
 func ClasspathExists(cp string) bool {
 	_, err := os.Stat(cp)
-	if err != nil {
-		return false
-	}
-	return true
+	return err == nil
 }

--- a/internal/app/packages/version_test.go
+++ b/internal/app/packages/version_test.go
@@ -67,21 +67,15 @@ func TestVersion_CopyToClassPath(t *testing.T) {
 }
 
 func TestVersion_GetFilename(t *testing.T) {
-	type fields struct {
-		Tag       string
-		Path      string
-		Algorithm string
-		CheckSum  string
-	}
 	tests := []struct {
-		name   string
+		name    string
 		version Version
-		want   string
+		want    string
 	}{
 		{
-			name: "Can Get Filename",
+			name:    "Can Get Filename",
 			version: driverV1,
-			want: "driver-0.0.1.txt",
+			want:    "driver-0.0.1.txt",
 		},
 	}
 	for _, tt := range tests {
@@ -100,12 +94,6 @@ func TestVersion_GetFilename(t *testing.T) {
 }
 
 func TestVersion_InClassPath(t *testing.T) {
-	type fields struct {
-		Tag       string
-		Path      string
-		Algorithm string
-		CheckSum  string
-	}
 	type args struct {
 		files []fs.FileInfo
 	}
@@ -113,16 +101,16 @@ func TestVersion_InClassPath(t *testing.T) {
 	var files, _ = ioutil.ReadDir(testPath + "/tests/mocks/installed")
 
 	tests := []struct {
-		name   string
+		name    string
 		version Version
-		args   args
-		want   bool
+		args    args
+		want    bool
 	}{
 		{
-			name: "Can Determine if Package in Classpath",
+			name:    "Can Determine if Package in Classpath",
 			version: extensionV2,
-			args: args{files},
-			want: true,
+			args:    args{files},
+			want:    true,
 		},
 	}
 	for _, tt := range tests {
@@ -141,21 +129,15 @@ func TestVersion_InClassPath(t *testing.T) {
 }
 
 func TestVersion_PathIsHttp(t *testing.T) {
-	type fields struct {
-		Tag       string
-		Path      string
-		Algorithm string
-		CheckSum  string
-	}
 	tests := []struct {
-		name   string
+		name    string
 		version Version
-		want   bool
+		want    bool
 	}{
 		{
-			name: "Can Determine if Path is Http",
+			name:    "Can Determine if Path is Http",
 			version: driverV2,
-			want: false,
+			want:    false,
 		},
 	}
 	for _, tt := range tests {
@@ -174,32 +156,26 @@ func TestVersion_PathIsHttp(t *testing.T) {
 }
 
 func TestVersion_calcChecksum(t *testing.T) {
-	type fields struct {
-		Tag       string
-		Path      string
-		Algorithm string
-		CheckSum  string
-	}
 	type args struct {
 		b []byte
 	}
 	tests := []struct {
-		name   string
+		name    string
 		version Version
-		args   args
-		want   string
+		args    args
+		want    string
 	}{
 		{
-			name: "Can Calc Checksum SHA1",
+			name:    "Can Calc Checksum SHA1",
 			version: driverV2,
-			args: args{[]byte("DriverSHA1")},
-			want: "70daefe06dd19c073920273e02cfc712951795ea",
+			args:    args{[]byte("DriverSHA1")},
+			want:    "70daefe06dd19c073920273e02cfc712951795ea",
 		},
 		{
-			name: "Can Calc Checksum SHA256",
+			name:    "Can Calc Checksum SHA256",
 			version: driverV1,
-			args: args{[]byte("DriverSHA256")},
-			want: "94c74ea180983e2ec16451fed233c9f6d3d47572133cae84a0adc7c9fd7e1dd4",
+			args:    args{[]byte("DriverSHA256")},
+			want:    "94c74ea180983e2ec16451fed233c9f6d3d47572133cae84a0adc7c9fd7e1dd4",
 		},
 	}
 	for _, tt := range tests {

--- a/tests/endtoend/help.yml
+++ b/tests/endtoend/help.yml
@@ -7,7 +7,7 @@ tests:
           - Search for, install, and uninstall liquibase drivers, extensions, and utilities.
           - lpm [command]
           - add         Add Packages
-          - completion  generate the autocompletion script for the specified shell
+          - completion  Generate the autocompletion script for the specified shell
           - help        Help about any command
           - install     Install Packages from liquibase.json
           - list        List Installed Packages

